lib/database_connection.rb

```diff
@@ -0,0 +1,15 @@
+require 'pg'
+
+class DatabaseConnection
+  def self.setup(dbname)
+    @connection = PG.connect(dbname: dbname)
+  end
+
+  def self.connection
+    @connection
+  end
```

Comment:
You can get rid of lines 8 - 10  and the test as well, as this gets tested inside the "query" method.